testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/TokenIntrospectionTest.java

```diff
@@ -471,6 +471,57 @@ public void testIntrospectionRequestParamsMoreThanOnce() throws Exception {
         assertEquals(OAuthErrorException.INVALID_REQUEST, errorRep.getError());
     }
 
+    @Test
+    public void testIntrospectRevokeRefreshToken() throws Exception {
```

Comment:
Both structures of `testIntrospectRevokeRefreshToken` and `testIntrospectRevokeOfflineToken` are similar. It might be good if duplicated codes are removed.

Reply:
@tnorimat 
I fixed. Could you confirm it?

Reply:
Thank you.